src/stories/dumb-components/datepicker.stories.ts

```diff
@@ -0,0 +1,50 @@
+import '@/scss/main.scss';
+import { ref } from 'vue';
+import OeDatepicker from '@components/dumb/OeDatepicker.vue';
+import type { Meta, StoryObj } from '@storybook/vue3';
+
+// More on how to set up stories at: https://storybook.js.org/docs/vue/writing-stories/introduction
+const meta: Meta<typeof OeDatepicker> = {
+  title: 'Dumb components/Datepicker',
+  component: OeDatepicker,
+  parameters: {
+    docs: {
+      description: {
+        component:
+          'Wrapper component for VlDatepicker.\n\n Component handles a date in API format (yyyy-MM-dd) and allows input using visual format (dd-MM-yyyy).',
+      },
+    },
+  },
+  tags: ['autodocs'],
+  argTypes: {
+    modelValue: {
+      description: 'Date in API format (yyyy-MM-dd)',
+      control: 'text',
+      table: {
+        type: { summary: 'string' },
+        defaultValue: { summary: null },
+      },
+    },
+  },
+};
```

Comment:
![image](https://github.com/user-attachments/assets/54ea10c1-1d22-4ecb-9918-ee60dcde1f92)
Deze geeft error als ge iets manueel wilt ingeven, niet zeker of het uitmaakt

Reply:
Storybook input triggered bij elke key-up, niet relevant voor datepicker. Control gedisabled.